.openapi.yaml

```diff
@@ -0,0 +1,337 @@
+openapi: 3.0.2
+info:
+  title: EployeeAPI
+  version: "1.0.0"
+
+paths:
+  /auth:
+    post:
+      security:
+        - bearerAuth: []
+      description: "Authorization user"
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              oneOf:
+                - $ref: "#/components/schemas/User"
+      responses:
+        '200':
+          description: "Successfully"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/responses/Success"
+        '401':
+          description: "Unauthorization"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/responses/UnauthorizedError"
+        '404':
+          description: "Page not found"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/responses/NotFoundError"
+        '500':
+          description: Enternal Server Error.
+  /employees:
+    get:
+      description: Get all employees
+      parameters:
+        - in: query
+          name: page
+          schema:
+            type: integer
+          description: Number of equal parts of employees
+        - in: query
+          name: number
+          schema:
+            type: integer
+          description: Number of element of one page
+      responses:
+        '200':
+          description: Get an array of employees
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
```

Comment:
Where is a field with data itself? 

Also, you have to return whether number of pages or total number of items to do pagination correctly.